pmlc/conversion/stdx_to_llvm/stdx_to_llvm.cc

```diff
@@ -352,14 +352,11 @@ struct FPToSILowering : public LLVMLegalizationPattern<stdx::FPToSIOp> {
   PatternMatchResult
   matchAndRewrite(Operation *op, ArrayRef<Value> operands,
                   ConversionPatternRewriter &rewriter) const override {
-    if (auto fptosiOp = cast<stdx::FPToSIOp>(op)) {
-      auto value = fptosiOp.getOperand();
-      auto stdxType = fptosiOp.getOperation()->getResult(0).getType();
-      auto llvmType = typeConverter.convertType(stdxType);
-      rewriter.replaceOpWithNewOp<LLVM::FPToSIOp>(op, llvmType, value);
-      return matchSuccess();
-    }
-    return matchFailure();
+    auto value = op->getOperand(0);
```

Comment:
If you want access to a particular post-converted operand, you can use the Adapter class, generated by ODS (i.e. tablegen).